product/views.py

```diff
@@ -1,3 +1,70 @@
-from django.shortcuts import render
+import json
+import re
+from django.http             import JsonResponse
+from django.views            import View
 
-# Create your views here.
+
+from product.models          import Product, Sale
+from user.models             import User
+from decorators.utils        import login_required
+
+import operator
+
+class ProductListView(View):  
+    # @login_required
+    def get(self,request, catCd, sorting):
```

Comment:
sorting 기능은 `path parameter`가 아니라 `query parameter`를 사용해서 구현해주세요~
```suggestion
    def get(self,request, catCd, sorting):
```

Reply:
감사합니다 쿼피 파라미터를 익혀서 다음 프로젝트때 더욱 유연하게 사용할수 있도록 연습해보겠습니다!